src/pages/payment/desktop/desktop.tsx

```diff
@@ -0,0 +1,69 @@
+import { TwoColumnCursor } from 'src/components/templates/two-column-cursor/two-column-cursor';
+import { JobDescrioptionCard } from 'src/components/templates/job-description-card';
+import { PaymentSummaryCard } from 'src/components/templates/payment-summary-card';
+import { PaymentMethods } from 'src/components/templates/payment-methods';
+import { Button } from 'src/components/atoms/button/button';
+import { BackLink } from 'src/components/molecules/back-link';
+import Dapp from 'src/dapp';
+import { usePaymentShared } from '../payment.shared';
+import css from './desktop.module.scss';
+
+export const Desktop: React.FC = () => {
+  const { offer, commision, total_price, start_date, isPaidCrypto, onClickProceedPayment, isDisabledProceedPayment } =
+    usePaymentShared();
+  const {
+    job_category: { name: job_name },
+    recipient: {
+      meta: { name: applicant_name, avatar, city, country },
+      type,
+    },
+    project: { payment_scheme },
+    total_hours,
+    assignment_total,
+  } = offer || {};
+
+  return (
+    <TwoColumnCursor>
+      <div className={`${css.container} ${css.right}`}>
+        <BackLink title="Return" onBack={() => history.back()} />
+        <JobDescrioptionCard
+          job_title={job_name}
+          start_date={start_date}
+          end_date="Present"
+          info_list={[
+            { icon: 'suitcase', name: payment_scheme },
+            { icon: 'hourglass', name: `${total_hours} hrs` },
+          ]}
+          img={avatar as string}
```

Comment:
I believe the avatar here could be `undefined` when the user is of type `users`. Please double check with Ehsan

Reply:
Done.